libs/core-ui/src/lib/components/InteractiveLegend.styles.ts

```diff
@@ -11,85 +11,125 @@ import {
-  () => {
-    const theme = getTheme();
-    return mergeStyleSets<IInteractiveLegendStyles>({
-      clickTarget: {
-        alignItems: "center",
-        cursor: "pointer",
-        display: "flex",
-        flex: "1",
-        flexDirection: "row"
-      },
-      colorBox: {
-        borderRadius: "6px",
-        cursor: "pointer",
-        display: "inline-block",
-        height: "12px",
-        margin: "11px 4px 11px 8px",
-        width: "12px"
-      },
-      deleteButton: {
-        color: theme.semanticColors.errorText,
-        display: "inline-block",
-        width: "16px"
-      },
-      disabledItem: {
-        alignItems: "center",
-        backgroundColor: theme.semanticColors.disabledBackground,
-        display: "flex",
-        flexDirection: "row",
-        height: "34px",
-        marginBottom: "1px"
-      },
-      editButton: {
-        color: theme.semanticColors.buttonText,
-        display: "inline-block",
-        width: "16px"
-      },
-      inactiveColorBox: {
-        borderRadius: "6px",
-        cursor: "pointer",
-        display: "inline-block",
-        height: "12px",
-        margin: "11px 4px 11px 8px",
-        opacity: 0.4,
-        width: "12px"
-      },
-      inactiveItem: {
-        alignItems: "center",
-        color: theme.semanticColors.primaryButtonTextDisabled,
-        display: "flex",
-        flexDirection: "row",
-        height: "34px",
-        marginBottom: "1px"
-      },
-      item: {
-        alignItems: "center",
-        display: "flex",
-        flexDirection: "row",
-        height: "34px",
-        marginBottom: "1px"
-      },
-      label: {
-        cursor: "pointer",
-        display: "inline-block",
-        flex: "1"
-      },
-      root: {
-        paddingBottom: "8px",
-        paddingTop: "8px"
-      }
-    });
-  };
+export const interactiveLegendStyles: (
+  activated?: boolean,
+  color?: string
+) => IProcessedStyleSet<IInteractiveLegendStyles> = (
+  activated?: boolean,
+  color?: string
+) => {
+  const theme = getTheme();
+  return mergeStyleSets<IInteractiveLegendStyles>({
+    circleColorBox: {
+      backgroundColor: color,
+      borderRadius: "6px",
+      cursor: "pointer",
+      display: "inline-block",
+      height: "12px",
+      margin: "11px 4px 11px 8px",
+      opacity: activated ? 1 : 0.4,
+      width: "12px"
+    },
+    clickTarget: {
+      alignItems: "center",
+      cursor: "pointer",
+      display: "flex",
+      flex: "1",
+      flexDirection: "row"
+    },
+    deleteButton: {
+      color: theme.semanticColors.errorText,
+      display: "inline-block",
+      width: "16px"
+    },
+    diamondColorBox: {
+      backgroundColor: color,
+      cursor: "pointer",
+      display: "inline-block",
+      height: "11px",
+      margin: "11px 4px 11px 8px",
+      opacity: activated ? 1 : 0.4,
+      transform: "rotate(45deg)",
+      width: "11px"
+    },
```

Comment:
Could you place shape definitions together?

Reply:
Could you please help elaborate what does this mean? Thanks!

Reply:
I think linting will cause these names to be sorted alphabetically. This is ok.